scripts/publish.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x -e
+
+# Usage: grep -v '^#' packages.txt | scripts/publish.sh
+
+# Publish the given packages.
+while read LINE; do
+    cargo publish -p $LINE
+done
```

Comment:
It would be nice to do the following:
* Add the documentation for what are the requirements to the packages. I think the dependencies need to be ordered. Also, the individual requirements of packages from https://doc.rust-lang.org/cargo/reference/publishing.html have to be satisfied. Anything else?
* Write that if unsure things can be tested.

Reply:
`packages.txt` is already self-documented